.github/scripts/coderabbit-autofix/generate-instructions.sh

```
@@ -0,0 +1,95 @@
+## REVIEW SUMMARY
+
+HEADER_EOF
+
+# Add review body
+if [[ -n "$REVIEW_BODY" ]]; then
+  echo "$REVIEW_BODY" >> "$OUTPUT_FILE"
+else
+  echo "_No review summary provided._" >> "$OUTPUT_FILE"
+fi
+
+echo "" >> "$OUTPUT_FILE"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)